CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+## v2.0.1 (Mar 29, 2021)
+
+### Bugfixes
+
+* Add checks on spaces before and after bytes
+  * This will now throw an exception when spaces are present like ' 1.2.3.4' or '1. 2.3.4' or '1.2.3.4 '.
+
+### Internal Changes
+
+* Rewrite byte parsing without using JS `parseInt()`
+* Avoid some useless memory allocations
+* New Mocha testing suite, thanks @kaoudis [#36](https://github.com/rs/node-netmask/pull/36)
+
+## v2.0.0 (Mar 19, 2021)
+
+### Breaking Change
+
+Previous API was treating IPs with less than for bytes as IP with a
+netmask of the size of the provided bytes (1=8, 2=16, 3=24) and was
+interpreting the IP as if it was completed with 0s on the right side.
+
+*Proper IP parsing for these is to consider missing bytes as being 0s on
+the left side.*
+
+Mask size is no longer infered by the number of bytes provided.
+
+This means that the input `216.240` will no longer be interpreted as `216.240.0.0/16`, but as `0.0.216.240/32`,
```

Comment:
The convention is actually to parse this as 216.0.0.240. The last commit fixes that.